libs/cli/build.gradle

```diff
@@ -11,9 +11,18 @@ apply plugin: 'elasticsearch.publish'
 dependencies {
   api 'net.sf.jopt-simple:jopt-simple:5.0.2'
   api project(':libs:elasticsearch-core')
+
+  testImplementation(project(":test:framework")) {
+    exclude group: 'org.elasticsearch', module: 'elasticsearch-cli'
+  }
+}
+
+tasks.named("test").configure {
+  // Disable security to force new behavior in JDK 22.
+  // If enabled, JDK 22 doesn't provide a console if redirected.
+  systemProperty 'tests.security.manager', 'false'
```

Comment:
We should be able to avoid a blanket disabling by adding `@WithoutSecurityManager` to the new test class.

Reply:
Great, thanks!